st2common/st2common/models/api/keyvalue.py

```diff
@@ -134,12 +133,12 @@ def from_model(cls, model, mask_secrets=True):
             doc['value'] = symmetric_decrypt(KeyValuePairAPI.crypto_key, model.value)
             encrypted = False
 
-        scope = getattr(model, 'scope', SYSTEM_SCOPE)
```

Comment:
Wait, this should still be `user` and `system`, we only use ``st2kv`` notation in templates, right?

Or am I missing something?